.github/workflows/promote-iso.yml

```
@@ -0,0 +1,394 @@
+      - name: Set up Homebrew
+        uses: Homebrew/actions/setup-homebrew@8f52032f943acddc1c5670f437b94e031e78afc0 # master
+
+      - name: Install rclone
+        run: brew install rclone
+
+      - name: Preflight checks
+        id: preflight
+        env:
+          GH_TOKEN: ${{ github.token }}
+          GH_REPO: ${{ github.repository }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:133:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)